files/galaxy/walle/test_object_store_conf.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
```

Comment:
Could you please adapt the one from dev for this test file? This is so similar to production configuration that it is going to show up when searching for use of the string `/mnt/scratch` in the code base

Reply:
Have just removed this - also now included in walle role.